src/Console/Application.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+/*
+ * This file is part of the webmozart/php-scoper package.
+ *
+ * (c) Bernhard Schussek <bschussek@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Webmozart\PhpScoper\Console;
+
+use Silly\Application as SillyApplication;
+
+class Application extends SillyApplication
+{
+    /**
+     * The version of the PHP-Scoper.
+     */
+    const VERSION = '@package_version@';
```

Comment:
I've left it as-is for the time being. It's reasonably complex to figure out given there's a couple of methods php-scoper might end up being installed. An exercise for another day.